public/index.js

```diff
@@ -0,0 +1,60 @@
+// document.addEventListener("DOMContentLoaded", function () {
+//     const submenuIcon = document.querySelector(".submenu-icon");
+//     const menuItem = submenuIcon.closest(".pages__item");
+
+//     submenuIcon.addEventListener("click", function (event) {
+//       event.stopPropagation(); // Щоб уникнути небажаного поширення
+//       menuItem.classList.toggle("open");
+
+//       // зміна іконки при відкритті/закритті submenu
+
+//       if (menuItem.classList.contains("open")) {
+//         submenuIcon.textContent = "keyboard_arrow_up";
+//       } else {
+//         submenuIcon.textContent = "keyboard_arrow_down";
+//       }
+//     });
+// });
```

Comment:
Why is this code commented
Remove it or add // TODO: with explanation why and when this code should be uncommented